tests/backend/test_threshold_measurement.py

```diff
@@ -81,3 +81,18 @@ def test_binary_outcome(self, setup_backend, pure):
             for i in range(num_modes):
                 assert meas_results[0][i] == 0 or meas_results[0][i] == 1
 
+    def test_single_mode_measurement(self, setup_backend, pure):
```

Comment:
maybe we could add some displacement to this test too?

Reply:
Yes! Maybe we could merge this PR first and the update the test in yours?